src/mscorlib/src/System/BitConverter.cs

```diff
@@ -446,6 +446,16 @@ unsafe public static double ToDouble (byte[] value, int startIndex)
         [SecuritySafeCritical]
         public static unsafe double Int64BitsToDouble(long value) {
             return *((double*)&value);
-        }                    
```

Comment:
This line is the 'removed' one. Please note that it has not actually been removed 😄